pkg/k8s/pods/pod.go

```diff
@@ -212,11 +213,58 @@ func Exists(podName, namespace string, c kubernetes.Interface) bool {
 	return pod.GetObjectMeta().GetDeletionTimestamp() == nil
 }
 
-//ContainerLogs returns the logs of a given container in a given pod
-func ContainerLogs(container string, pod *apiv1.Pod, namespace string, c kubernetes.Interface) (string, error) {
+//GetDevPodUserID returns the user id running the dev pod
+func GetDevPodUserID(ctx context.Context, dev *model.Dev, c *kubernetes.Clientset) int64 {
+	devPodLogs, err := GetDevPodLogs(ctx, dev, false, c)
+	if err != nil {
+		log.Errorf("failed to access development environment logs: %s", err)
+		return -1
+	}
+	return parseUserID(devPodLogs)
+}
+
+func parseUserID(output string) int64 {
+	lines := strings.Split(output, "\n")
+	for _, line := range lines {
```

Comment:
this depends on `start.sh` logging a specific thing. Since that's in a different repo, it's going to be hard to keep them in sync. Could we add a test or something so that if we  remove that log by accident our CI will catch it?

Reply:
I think we should move the bin image to this repo, and bin-cloud to okteto app